fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2606,6 +2606,11 @@ public class Config extends ConfigBase {
     //==========================================================================
     //                    begin of cloud config
     //==========================================================================
+    @ConfField(description = {"是否启用FE 日志文件按照大小删除策略，当日志大小超过指定大小，删除相关的log。默认关闭，为按照时间策略删除",
+        "Whether to enable the FE log file deletion policy based on size, "
+            + "where logs exceeding the specified size are deleted. "
+            + "It is disabled by default and follows a time-based deletion policy."})
+    public static boolean log_by_size = false;
```

Comment:
Better use a enum type like:
`public static String log_rollover_strategy = "age"`

The options are `size` or `age`.

And don't forget to add the `option` field of `@ConfField`